cmd/mt-whisper-importer-reader/conversion_test.go

```diff
@@ -0,0 +1,1204 @@
+package main
+
+import (
+	"testing"
+
+	"github.com/kisielk/whisper-go/whisper"
+)
+
+func testIncResolution(t *testing.T, inData []whisper.Point, expectedResult map[string][]whisper.Point, method string, inRes, outRes, rawRes uint32) {
+	outData := incResolution(inData, method, inRes, outRes, rawRes)
+
+	if len(expectedResult) != len(outData) {
+		t.Fatalf("Generated data is not as expected:\n%+v\n%+v", outData, expectedResult)
```

Comment:
when this dumps both structures, it's not clear which is the one that was expected, which is what we one we got.
maybe better something like
```
t.Fatalf("testIncResolution.\nExpected:\n%+v\nGot:\n%+v\n", outData, expectedResult)
```

Reply:
ditto for the other ones btw

Reply:
Fixed